tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php

```diff
@@ -105,7 +105,16 @@ public function getFieldDefinition(string $field_name);
   public function getStoredTypes();
 
   /**
-   * Returns a list of property values for stored types..
+   * Returns a list of property types that should not be stored.
+   * This is the complement function of getStoredTypes.
```

Comment:
Make it more clear that by unstored here we mean that the value is not stored in the Drupal field tables. The property type does still have a column in the drupal field tables which is why we need to know this.


```suggestion
   * Returns a list of property types whose value should not be stored in Drupal
   * field tables.
   *
   * This is the inverse of the getStoredTypes() method. It's needed because all
   * field property types have a column in the Drupal table. However, the "Stored"
   * ones save the value both in Drupal and in the storage backend and the
   * "unstored" ones save an empty value in Drupal that is populated on load by
   * the storage backend.
```